pkg/executor/default.go

```diff
@@ -189,9 +189,10 @@ func (e *DefaultExecutor) dirOps(stage, dir string, fs vfs.FS, console plugins.C
 
 			config, err := schema.Load(path, fs, schema.FromFile, e.modifier)
 			if err != nil {
-				return err
-
+				e.logger.Warnf("failed to load file '%s': %s", path, err.Error())
+				return nil
```

Comment:
What if we try to so something like the following:

Do not return error on a schema loading failure, but keep the errors in a slice of a wider scope as the one step above method. Probably with a simple flag to notify something failed to load would be enough. Then after the walk over directories we just need to check if there were loading errors and return the result together with the loading error if any.

Reply:
Sounds like a good approach, I will make the modifications!

Reply:
I pushed commit 384d60e0329325f7f3ccb3b794858953f2ebd3bc to address this.